embedded-files/fuseml-core-deployment.yaml

```diff
@@ -21,6 +21,18 @@ roleRef:
   kind: ClusterRole
   name: cluster-admin
 ---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: badger
```

Comment:
could you call it `fuseml-core` or `fuseml` or `fuseml-data` pls ? badger is just a library in fuseml-core. Besides, other back-ends may be needing the PVC.

Reply:
right, that's why I actually call the volume differently already. But sure, I'll rename PVC top